fetch.bs

```diff
@@ -3348,6 +3396,8 @@ the request.
  <li><p>Run <a>main fetch</a> given <var>fetchParams</var>.
 </ol>
 
+<h3 id="sec-current-time">Current Time</h3>
+The <dfn>current time</dfn> is the present time given the <a for=/>global monotonic clock</a>.
```

Comment:
Yeah, we definitely want this to point at https://w3c.github.io/hr-time/#dfn-current-high-resolution-time

Reply:
See above, we need a new definition as we don't have a `current global` and we want to use the `global monotonic clock` and not rely on a `time origin`. But this new definition should maybe be a PR to `HR-TIME`.